conanfile.py

```diff
@@ -29,7 +29,7 @@ class CnlConan(ConanFile):
         "target": None,
         "test_pattern": None,
     }
-    generators = "cmake_find_package"
+    generators = "cmake_paths", "cmake_find_package"
```

Comment:
I spent a long time flip-flopping between _cmake_paths_ and _cmake_find_package_ but eventually found out how to make the latter work. I'm hopeful -- especially given I have an OSX pipeline job -- that _cmake_find_package_ can be sufficient.